cpp/src/binaryop/jit/code/operation.cpp

```diff
@@ -221,6 +221,14 @@ const char* operation =
             double y1 = static_cast<double>(y);
             return fmod(fmod(x1, y1) + y1, y1);
         }
+
+        template <typename TypeOut,
```

Comment:
you will need to add a similar method to `RPymod` type as well to operate between a scalar and a column.

Reply:
Thanks @sriramch, added similar method to `RPymod`